### PR TITLE
First translation from French to English

### PR DESCRIPTION
**Pull request by mlep**

### CODE REVIEW

File: en/User/Organize_articles.md [hunk 1]
```diff
@@ -5,5 +5,66 @@ subTitle: Organize articles
...
+To be able to find more easily your articles, several methods are available. 
 
-To be translated from [Classer ses articles](../../fr/Utilisateur/Classer_ses_articles.md).
\ No newline at end of file
+
+## Tags
+### Assign a tag
+#### From the article
+
+Currently, it is not yet possible to assign a tag while saving a new article.
```
Comment (tcitworld) [inline en/User/Organize_articles.md]: This has to be corrected, french doc is not up to date : https://www.wallabag.org/blog/2015/02/18/wallabag-1-9-many-features

Comment (tcitworld) [inline en/User/Organize_articles.md]: Now possible to tag an article from web interface while saving.

File: en/User/Organize_articles.md [hunk 2]
```diff
@@ -5,5 +5,66 @@ subTitle: Organize articles
...
+
+Hence, the add of tag is done on already-saved article.
+
+On the reading page of the article, there is a part `tags` followed with a pencil. Click on the pencil.
+
+Input in the box the tags you want to assign. Add as many tags as you want. You can separate them with a comma to save some time. Auto-completion is available here: when you type the first letters of an existing tag, it is not necessary to type the whole word. Select it in the drop-down list. Once all your tags are inputted, click  on the button **Tag**.
```
Comment (tcitworld) [inline en/User/Organize_articles.md]: > You can separate them with a comma to save some time.

I don't know why french has this, but it's kind of an odd sentence.

Comment (mlep) [inline en/User/Organize_articles.md]: I have no idea how to solve this...

Comment (tcitworld) [inline en/User/Organize_articles.md]: Just say something like « They need to be separated by commas and not spaces »

Comment (mlep) [inline en/User/Organize_articles.md]: Thanks. Done. I am going to do it now in the French localization.

**Pull request merged**